### DISCUSSION TITLE
cPanel: How to update version?

### DISCUSSION THREAD

**Discussion by frank-github** [FreshRSS/FreshRSS#6935] [Q&A] [answered]
### Describe the bug

no bug  :-)

My WebHoster has an easy way to install FreshRSS with only few clicks (cPanel) :-)
but he do not give the option to update to new versions of FreshRSS :-1: 

I have access to an terminal over ssh and can upload the latest freshRSS-zip
But what the correct steps to update my installation with this zip file?

### To Reproduce

nothing

### Expected behavior

description, how to manually update installation

### FreshRSS version

1.24.1

### Environment information

Web-Hoster

**Comment by Alkarex** [comment]
I assume the built-in Web-based update did not work for you?
If you have SSH and the git command, try https://github.com/FreshRSS/FreshRSS/discussions/6769

**Comment by frank-github** [comment, marked as answer]
ohh, my god, what a blame 
I did not look as admin logged in :-(

As Admin I found web updater and worked smooth :+1:
